File: Python/tests/test_bart_rl.py
import pytest

from hbayesdm.models import bart_rl


def test_bart_rl():
    _ = bart_rl(
        data="example", niter=10, nwarmup=5, nchain=1, ncore=1)


if __name__ == '__main__':
    pytest.main()

File: R/tests/testthat/test_bart_rl.R
context("Test bart_rl")
library(hBayesDM)

test_that("Test bart_rl", {
  # Do not run this test on CRAN
  skip_on_cran()

  expect_output(bart_rl(
      data = "example", niter = 10, nwarmup = 5, nchain = 1, ncore = 1))
})
